docs/docsite/rst/mazer/examples.rst

```diff
@@ -10,468 +10,346 @@ Examples
 
 This topic provides examples for using Mazer CLI.
 
-Installing Roles
-----------------
+.. _installing_collections:
 
-To install a role found on `Galaxy <https://galaxy.ansible.com>`_, pass the *namespace.role_name*. For example,
-the following install the role `geerlingguy.nginx <https://galaxy.ansible.com/geerlingguy/nginx/>`_ from the
-Galaxy server:
+Installing Collections
+----------------------
 
-.. code-block:: bash
-
-    $ mazer install geerlingguy.nginx
-
-.. note::
-
-    Before installing roles with Mazer, review :ref:`using_mazer_content`. Mazer installs content different from
-    the way ``ansible-galaxy`` does.
-
-To install a specific version of the Galaxy role, append `,<semantic version>` to the name, as demonstrated by the
-following:
-
-.. code-block:: bash
-
-    $ mazer install geerlingguy.nginx,2.6.0
-
-Roles can also be installed directly from GitHub, by passing the git URL of the repository, and a namespace value. Use
-the required *--namespace* option to provide the namespace. The following installs the same role, but bypasses the Galaxy server,
-and goes directly to GitHub.
-
-.. code-block:: bash
-
-    $ mazer install --namespace geerlingguy git+https://github.com/geerlingguy/ansible-role-nginx
-
-A version number can also be passed using the same ',' separated format. For example:
-
-.. code-block:: bash
-
-    $ mazer install git+https://github.com/geerlingguy/ansible-role-nginx,2.6.0
+Collections are a new way to package and distribute ansible related content.
+See :ref:`creating_collections` for examples.
 
+Collections are available starting with version 3.2 of the Galaxy server and version 2.9 of ansible.
```

Comment:
2.9 should be 2.8, even though its "tech preview" we do say 2.8 elsewhere and 2.9 won't be available yet.